modules/Bio/EnsEMBL/Compara/RunnableDB/HomologyAnnotation/ParsePAFforBHs.pm

```diff
@@ -0,0 +1,121 @@
+=head1 LICENSE
+
+See the NOTICE file distributed with this work for additional information
+regarding copyright ownership.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+=head1 NAME
+
+Bio::EnsEMBL::Compara::RunnableDB::HomologyAnnotation::ParsePAFforBHs
+
+=head1 DESCRIPTION
+
+=cut
+
+package Bio::EnsEMBL::Compara::RunnableDB::HomologyAnnotation::ParsePAFforBHs;
+
+use warnings;
+use strict;
+use Bio::EnsEMBL::Registry;
+use Bio::EnsEMBL::Compara::DBSQL::DBAdaptor;
+use Bio::EnsEMBL::Compara::PeptideAlignFeature;
+use List::MoreUtils qw/uniq/;
+use Data::Dumper;
+
+use base ('Bio::EnsEMBL::Compara::RunnableDB::BaseRunnable');
+
+sub run {
+    my $self = shift;
+
+    my $query_gdb_id   = $self->param_required('genome_db_id');
+    my $hit_gdb_id     = $self->param_required('target_genome_db_id');
+    my $seq_member_ids = $self->param('member_id_list');
+
+    my $paf_adaptor    = $self->compara_dba->get_PeptideAlignFeatureAdaptor;
+    my $ref_db         = Bio::EnsEMBL::Compara::DBSQL::DBAdaptor->go_figure_compara_dba($self->param_required('rr_ref_db'));
+    my $ref_mem_adap   = $ref_db->get_SeqMemberAdaptor;
+    my $query_mem_adap = $self->compara_dba->get_SeqMemberAdaptor;
+
+    # mlss will not exist for hit_gdb is reference and belongs to a different db
```

Comment:
Agreed, no idea what I was trying to write, I think it may be a case of accidentally touching mousepad whilst typing and partially erasing something :thinking: